### ISSUE TITLE
AP file

### ISSUE THREAD

**Issue by GiovanniVolta** [XENONnT/WFSim#128]
https://github.com/XENONnT/WFSim/blob/9de9ebad2706a8fd3f2cab4dc9d7c68cdacfa6ca/wfsim/load_resource.py#L171

Hi all, 
the AP file in 'private_nt_aux_files/sim_files/' is a json file, and straxen.get_resource is trying to open a pkl. Is it on purpose?

I am trying to make simulation using some instructuions. I am then defing the context liket this:

```
conf = straxen.contexts.xnt_common_config
conf['gain_model'] = ('to_pe_per_run', '/home/gvolta/XENONnT/private_nt_aux_files/sim_files/to_pe_nt.npy')
st   = strax.Context(storage='/dali/lgrandi/giovo/XENONnT/WFsim_data/strax_data/',
                     config=dict(detector='XENONnT',
                                 fax_config='/home/gvolta/XENONnT/private_nt_aux_files/sim_files/fax_config_nt_low_field.json',
                                 fax_file=f'/home/gvolta/XENONnT/Analysis/WFSim/instructions/{run_id}.csv',
                                 **conf,
                                 fax_config_override={
                                     'electron_extraction_yield': 1,  # temp fix
                                     's2_secondary_sc_gain': 31.5 / 0.09091864874478181,  # temp fix
                                     'pmt_ap_t_modifier': 0,
                                     'url_base': '/home/gvolta/XENONnT/private_nt_aux_files/sim_files/',
                                     #'photon_area_distribution': 'XENONnT_spe_distributions_20210305.csv',
                                     #'photon_ap_cdfs':'XENONnT_pmt_afterpulse_config_012605.json.gz',
                                     'enable_pmt_afterpulses':False,
                                 }),
                     **straxen.contexts.common_opts)
st.register(wfsim.RawRecordsFromFaxNT)
```

**enable_pmt_afterpulses** must be False otherwise I got an error

**Comment by petergaemers** [comment]
Ah this is annoying. The old file is a pkl.gz, the new one a json

**Comment by zhut19** [comment]
Thanks @GiovanniVolta and @petergaemers , should be fixed by merging #117
